app.js

```diff
@@ -127,9 +127,20 @@ if (Configuration.get('FEATURE_MORGAN_LOG_ENABLED')) {
 app.use(bodyParser.json({ limit: '10mb' }));
 app.use(bodyParser.urlencoded({ extended: true, limit: '10mb' }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, `build/${themeName}`)));
+
 app.use('/locales', express.static(path.join(__dirname, 'locales')));
 
+let themeAssetDir = path.join(__dirname, `build/${themeName}`);
+if (!fs.existsSync(themeAssetDir)) {
+	// fallback to static dir, for example for test env
+	themeAssetDir = path.join(__dirname, 'static');
+}
+const staticify = require('staticify')(themeAssetDir, { maxAgeNonHashed: '1d' });
+app.use(staticify.middleware);
+
+// backup
+app.use(express.static(path.join(__dirname, `build/${themeName}`)));
```

Comment:
keep this line where it was before

Reply:
actually this is not really need and it has to come after the staticify habnlder, otherwise the fallback cache for assets is not working